### ISSUE TITLE
Get list of files to do custom upload logic

### ISSUE THREAD

**Issue by alejost848** [meistudioli/msc-image-uploader#1]
Hello, thanks for creating this component. UX wise is perfect for what I'm working on. I was wondering if there was a way to get the list of files exposed as a property or as part of the `msc-image-uploader-pick` event. I'm looking to implement Firebase Storage to handle uploads but I'm not sure how to do it using this component. Any help would be appreciated.

**Comment by meistudioli** [comment]
No. the only way is through property `uploadInfo`, but this should be data from webservice.
